### PR TITLE
Alias fix up

### PR DESCRIPTION
**Pull request by znicholls**
Turns out that we had accidentally assigned "yr" to be an alias for "a". This meant we had the following behaviour

```python
>>> from openscm_units import unit_registry
>>> val = unit_registry.Quantity(1, "yr")
>>> val
<Quantity(1, 'a')>
```

This is quite annoying if you need to use string comparisons later (e.g. because you're passing to Fortran).

This PR fixes this so that if you pass in "yr", it stays as yr i.e. you get

```python
>>> from openscm_units import unit_registry
>>> val = unit_registry.Quantity(1, "yr")
>>> val
<Quantity(1, 'yr')>
```

### CODE REVIEW

**Comment by znicholls** [comment]
Surprisingly subtle. Pint's API features once again showing some of their, arguably, less desirable side.

**Comment by znicholls** [comment]
@crdanielbusch @JGuetschow might be of interest for you too as you have to deal with pint sometimes too

File: src/openscm_units/_unit_registry.py [hunk 1]
```diff
@@ -232,7 +232,8 @@ def add_standards(self) -> None:
 
         self._add_gases({x: x for x in MIXTURES})
 
-        self.define("a = 1 * year = annum = yr")
+        self.define("yr = 1 * year")
+        self.define("a = 1 * year = annum")
```
Comment (znicholls) [inline src/openscm_units/_unit_registry.py:236]: Very subtle change. It wasn't obvious to me straight away that 'everything on the right is an aliases for the first thing on the left'

**Review by mikapfl — APPROVED** [review]
> This is quite annoying if you need to use string comparisons later (e.g. because you're passing to Fortran).

I don't understand the use-case (don't you have to convert to something explicitly anyway if you need a specific unit string?), but the functionality works as advertised and doesn't seem to break anything.

**Comment by znicholls** [comment]
> I don't understand the use-case (don't you have to convert to something explicitly anyway if you need a specific unit string?), but the functionality works as advertised and doesn't seem to break anything.

Bit hard to explain. Let me try again. We pass stuff from Python to Fortran. In Fortran, there is no pint, so we just carry the units round as plain strings. To check that the units are as expected, we just do a basic string comparison. When pint magically converts "yr" to "a", our Fortran-side string comparisons break because they're expecting "yr" but getting "a". We could just change all our Fortran to expect "a", but this magical change feels wrong so I'm trying this instead.

**Comment by mikapfl** [comment]
But couldn't you run `thing.to("unit / yr")` before passing `thing` from python to Fortran?

**Comment by znicholls** [comment]
> But couldn't you run thing.to("unit / yr") before passing thing from python to Fortran?

In some cases, but in this case no (can explain on a call one day if you want to dig in further, I'm going to merge as is now though because the change is minor as you say).
